content/installation/installing-manager.md

```diff
@@ -93,16 +93,34 @@ After you install Cloudify Manager, you can change the settings used by the inst
 
 ### Additional Cloudify Manager Settings
 
-In addition to the command line options, the ```/etc/cloudify/config.yaml``` ([View in GitHub](https://github.com/cloudify-cosmo/cloudify-manager-install)) contains more advanced configuration settings, including:
+In addition to the command line options, the ```/etc/cloudify/config.yaml``` ([View in GitHub](https://github.com/cloudify-cosmo/cloudify-manager-install/blob/master/config.yaml)) contains more advanced configuration settings, including:
 
 * Administrator password
 * Private and public IP addresses
 * External REST communications over HTTPS
 * Local path replacement for remote resources with a URL
-* Management networks for Cloudify agents
+* Multi-network management
 * LDAP connection information
 * SSL communication settings
 
+#### Multiple Networks
+
+If a manager has a private IP (1.2.3.4) and two additional IPs (10.0.0.1 and 192.168.0.2) through which an agent can connect to it, each IP must be specified in the config.yaml. In the blueprint the agent can be configured to use one of these networks. If no IP address is specified in the blueprint, the agent connects to the private IP (1.2.3.4).
```

Comment:
No need to differentiate between private and additional IPs. They are all private IPs, just on different interfaces